http_check/check.py

```diff
@@ -319,6 +320,14 @@ def send_status_down(loginfo, message):
             else:
                 send_status_up("%s is UP" % addr)
 
+        # Report status as a metric as well
+        can_status = 1 if service_checks[0][1] == "UP" else 0
```

Comment:
I can do that. I first want to mention that my thinking/philosophy would be to separate calculating status from reporting status. A couple points in favor of that approach are:

* the logic to evaluate status isn't cluttered with call(s) to report the status
* the code to report status would be in exactly one place

I want to make sure I understand your request. Where in the code are you suggesting I send the metrics? In the `send_status_up` and `send_status_down` methods? I think those methods only append to `service_checks[]`. They don't actually send anything to the daemon or api. They also aren't used consistently, for example when catching a timeout exception around line 248.

Thanks,
Stephen